src/Content/Changes.php

```diff
@@ -62,6 +69,18 @@ public function files(): Files
 		return $files;
 	}
 
+	/**
+	 * Rebuilds the cache by finding all models with changes version
+	 */
+	public function generateCache(): void
+	{
+		foreach ($this->kirby->models() as $model) {
+			if ($model->version(VersionId::changes())->exists('*') === true) {
+				$this->track($model);
```

Comment:
@bastianallgeier Looking at this once more, this might be quite inefficient to call `::track()` for each model. Maybe this method should rather collect three arrays (files, pages, users) and then write them each with `::update()` once?

Reply:
Yeah, that makes sense.